style.css

```diff
@@ -0,0 +1,60 @@
+.nav-bar {
```

Comment:
pay attention to requirements "Take into consideration how you will structure your css + use BEM naming conventions",
you should avoid having a single css file for storing you styles, think of common elements and page specific 
(for example for the header I should create a header.css file and I would put all the styles for the header there)
+think about re-usability, if there are common elements that you would like to reuse in more pages, you should only style them in one place